src/main/java/seedu/parser/IndexParser.java

```diff
@@ -0,0 +1,31 @@
+package seedu.parser;
+
+import seedu.contact.Contact;
+import seedu.contact.ContactList;
+import seedu.exception.MissingArgException;
+
+
+public class IndexParser {
+    private static final int COMD_WORD_LENGTH = 1;
+    private static final int INDEX_POSITION = 1;
+    private static final int SIGNIFICANT_INDEX_POSITION = 0;
+    private static final int COMD_INDEX_LENGTH = 2;
+
+
+    public static int getIndexFromInput(String userInput, String command)
+            throws NumberFormatException, MissingArgException {
+        String[] comdIndexSplit = userInput.split(" ", COMD_INDEX_LENGTH);
+        if (comdIndexSplit.length == COMD_WORD_LENGTH) {
```

Comment:
Would it be better if the if block was changed to `comdIndexSplit.length <= COMD_WORD_LENGTH)` to catch length 0? Not entirely sure if the length might ever be zero but if so it probably wouldn't hurt to cover that case.

Reply:
Thank you, boss man, will do.